### ISSUE TITLE
1.8.7 and 1.8.8 is broken

### ISSUE THREAD

**Issue by gisu** [vue-styleguidist/vue-styleguidist#202]
When I install the latest version into a project, I cannot call the styleguidist, because Node does not find the executable.

```bash
> vue-styleguidist server

env: node\r: No such file or directory
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! testme@1.0.0 styleguide: `vue-styleguidist server`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the testme@1.0.0 styleguide script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

When I downgrade to 1.8.6 I can start Styleguidist again. Is there a new command or are the new versions just broken?

**Comment by elevatebart** [comment]
Hello @gisu, sorry you had this issue.

Do you mind sharing what OS/version of node/version of npm you are using ?

I never type `vue-styleguidist server` myself. 
I have an entry in the script portion of my package.json saying
```js
scripts: {
  "styleguide": "styleguidist server"
}
```
notice that there is no `vue-` before the `styleguidist` here.

**Comment by elevatebart** [comment]
I am surprised you get it working with 1.8.6 though. I will check the changelog.

**Comment by elevatebart** [comment]
How did you install vue-styleguidist?

**Comment by elevatebart** [comment]
I think I see it now:
```bash
> vue-styleguidist server

env: node\r: No such file or directory
```

the \r sould not be here

**Comment by elevatebart** [comment]
It should be fixed in v1.8.9

**Comment by elevatebart** [comment]
hey @gisu is it fixed now ? can I close this issue ?
